src/PavValHackathon.Web.API/Migrations/20210424103829_InitialCreate.Designer.cs

```diff
@@ -0,0 +1,25 @@
+﻿// <auto-generated />
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using PavValHackathon.Web.Data.Contexts;
+
+namespace PavValHackathon.Web.API.Migrations
+{
+    [DbContext(typeof(DataContext))]
+    [Migration("20210424103829_InitialCreate")]
+    partial class InitialCreate
+    {
+        protected override void BuildTargetModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
```

Comment:
lewl 1st commit disabling warnings

Reply:
in auto generated code ROFL

Reply:
honestly, I have no clue why a code generator ignores an `obsolete` warning.